src/app/devices/deviceContent/components/deviceContentNav.tsx

```diff
@@ -26,7 +26,7 @@ interface DeviceContentNavState {
 }
 
 export const NAV_LINK_ITEMS_PNP = ['interfaces', 'settings', 'properties', 'commands', 'events'];
-export const NAV_LINK_ITEMS_NONPNP = ['identity', 'twin', 'events'];
+export const NAV_LINK_ITEMS_NONPNP = ['identity', 'twin', 'events', 'methods'];
```

Comment:
snapshot test of this component needs to be updated accordingly. But before we do so, I'd like to add the labels at least on the deviceMethod before we enable it in the route as user would see it in the next release if we enable it

Reply:
Yeah I'm fixing the tests now and going to revert the methods change. That shouldn't go in this PR.